GCTC-NTGC/gc-digital-talent
# a11y - 3831 - Increase Colour Contrast

Resolves #3831 

## Summary

This darkens the colours for some `react-intl` tags and the select fields to meet the 4.5:1 colour contrast [outlined in WCAG ](https://www.w3.org/TR/WCAG21/#contrast-minimum).

## Screenshots

### Heavy Primary

<img width="548" alt="Screen Shot 2022-09-27 at 9 48 50 AM" src="https://user-images.githubusercontent.com/4127998/192544528-a6667c4d-6655-4580-8b75-2f587e64a73b.png">

### Red

<img width="405" alt="Screen Shot 2022-09-27 at 9 49 21 AM" src="https://user-images.githubusercontent.com/4127998/192544670-d7766174-ae9e-4c31-b752-4f1d0476d946.png">

### Select

<img width="572" alt="Screen Shot 2022-09-27 at 9 49 49 AM" src="https://user-images.githubusercontent.com/4127998/192544785-39787543-39c4-4638-a686-6fc70b761f46.png">

## Testing

1. Build all projects `npm run production --workspaces`
2. Navigate to some places these tags are used
3. Run axe dev tools and confirm there are no colour contrast errors

> **Note**
> There are some false positives, specifically titles that appear on images/gradients. This is because axe has trouble identify colour contrast when either the foreground or background include multiple colours.
